packages/test-utils/src/reporter.ts

```diff
@@ -1256,7 +1260,9 @@ export default class ReplayReporter<
       numUploaded,
     });
 
-    log(output.join("\n"));
+    if (output.length > 0) {
+      log(output.join("\n"));
+    }
```

Comment:
This was a pet peeve because it used to log empty lines like
```
[replay.io]: 🕑 Completing some outstanding work ...
[replay.io]:
```

Reply:
❤️ 

I wonder if we could improve the "Completing some outstanding work" message. We don't want to make it too specific so it's kinda nice that it's vague... but it always felt off to me. It feels very... formal/uptight/idontknowhowtocallit. Maybe it's just my non-english brain though.

Reply:
Not just your brain. I've never loved this message either.

Reply:
I dunno maybe this is better

```
  2 passed (9.0s)
[replay.io]: Finishing up. This should only take a moment ...
[replay.io]: 
[replay.io]: 🚀 Successfully uploaded 1 recordings:
[replay.io]: 
[replay.io]:    ✅ login-logout-library: should login and logout from the library
[replay.io]:       https://app.replay.io/recording/8a18b945-7842-4505-b8aa-c4537c28b908
```

Reply:
Much better!